### PR TITLE
Pages 4.0 Clean Up

### LINKED ISSUE #27283: Improvements to Pages 4.0
- Use SkipForRepeat's new EE Or Later
- Use lowercase for tested.features 
- Review .setttings to be consistent with other EE11 projects
- Formatting JSP test pages

See comments found in PR https://github.com/OpenLiberty/open-liberty/pull/27114

- [x] Formatting JSPs under Pages40Changes.war
- [x]  Remove trace from pages40Server
- [x] Update servlet api in dev/io.openliberty.pages.4.0.internal_fat/bnd.bnd
- [x] Add comment to GenerateTagFileVisitor
- [x] Consistent .settings 
- [x] Use EE OR LATER in MiscTests & JSP23JSP22ServerTest
- [x] Use lowercase in tested.features in 2.3 fat

### PR DESCRIPTION
**Pull request by volosied**
fixes https://github.com/OpenLiberty/open-liberty/issues/27283

### CODE REVIEW

File: dev/io.openliberty.pages.4.0.internal.factories/.settings/org.eclipse.core.resources.prefs [hunk 1]
```diff
@@ -1,2 +1,4 @@
+#Ant properties
+#Automatically generated by the ant prepare.settings.files task
 eclipse.preferences.version=1
 encoding/<project>=UTF-8
```
Comment (volosied) [inline dev/io.openliberty.pages.4.0.internal.factories/.settings/org.eclipse.core.resources.prefs:4]: Matches EL 6.0 --  I used: 

`cp -r io.openliberty.jakarta.expressionLanguage.6.0/.settings io.openliberty.pages.4.0.internal.factories`

File: dev/io.openliberty.pages.4.0.internal.factories/.settings/org.eclipse.jdt.ui.prefs [hunk 2]
```diff
@@ -1,116 +1,60 @@
-#Ant properties
```
Comment (volosied) [inline dev/io.openliberty.pages.4.0.internal.factories/.settings/org.eclipse.jdt.ui.prefs:1]: Copied via `cp -r io.openliberty.org.apache.jasper.expressionLanguage.6.0/.settings io.openliberty.pages.4.0.internal.factories`

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.pages.3.1.internal_fat,io.openliberty.pages.4.0.internal_fat,com.ibm.ws.jsp.2.3_fat

File: dev/io.openliberty.pages.4.0.internal_fat/publish/servers/pages40Server/bootstrap.properties [hunk 3]
```diff
@@ -9,4 +9,3 @@
 ###############################################################################
 bootstrap.include=../testports.properties
 osgi.console=7777
-com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/publish/servers/pages40Server/bootstrap.properties:12]: Let's update the copyright to include 2024 now

File: dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/testQueryStringonError.jsp [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2023 IBM Corporation and others.
+    Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/testQueryStringonError.jsp:2]: Just formatting we don't need to change a copyright.

File: dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/jspPlugin.jsp [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2023 IBM Corporation and others.
+    Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/jspPlugin.jsp]: If the only change is formatting no need to update

File: dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/jspParams.jsp [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2023 IBM Corporation and others.
+    Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/jspParams.jsp]: If the only change is formatting no need to update

File: dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/jspFallback.jsp [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2023 IBM Corporation and others.
+    Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/jspFallback.jsp]: If only formatting no need to update.

File: dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/isThreadSafe.jsp [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2023 IBM Corporation and others.
+    Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/isThreadSafe.jsp]: NO need to update if only changing formatting

File: dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/testQueryStringonError.jsp [hunk 9]
```diff
@@ -8,14 +8,16 @@
     SPDX-License-Identifier: EPL-2.0
  -->
 <html>
-<title>testQueryStringonError</title>
-<body>
+    <head>
+        <title>testQueryStringonError</title>
+    </head>
+    <body>
```
Comment (pnicolucci) [inline dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/testQueryStringonError.jsp:14]: You added the <head> element so copyright update is ok!

**Comment by volosied** [comment]
#libby

**Review by helyarp — COMMENTED** [review]
Looks good

**Review by helyarp — APPROVED** [review]
Adding Approve

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#libby

**Pull request merged**